pandas-stubs/core/arrays/numpy_.pyi

```diff
@@ -5,24 +5,16 @@ from pandas.core.arrays.base import (
     ExtensionOpsMixin,
 )
 
+from pandas._typing import npt
+
 from pandas.core.dtypes.dtypes import ExtensionDtype
 
 class PandasDtype(ExtensionDtype):
-    def __init__(self, dtype) -> None: ...
-    @property
-    def numpy_dtype(self): ...
-    @property
-    def name(self): ...
-    @property
-    def type(self): ...
-    @classmethod
-    def construct_from_string(cls, string): ...
-    @classmethod
-    def construct_array_type(cls): ...
+    def __init__(self, dtype: npt.DTypeLike) -> None: ...
```

Comment:
This is not documented, so I don't think we should include it in typing

Reply:
It is used to provide inheritance for other types, e.g., `class Int8Dtype(_IntegerDtype)`.